refget.md

```diff
@@ -368,16 +378,40 @@ Array of strings listing the versions of this API supported by the server.
 GET /sequence/service-info
 
```

Comment:
Shouldn't we take the opportunity of a major version change to move the service-info at the top level ?
I think this is what's implied in the [explanation](https://github.com/ga4gh-discovery/ga4gh-service-info#how-do-i-describe-a-service-implementing-multiple-specifications) of how multiple services should be described.

Reply:
Done in commit 12cbba1ceaed8dd71307a9ae71e397e0ce2d690e